cypress/e2e/layout/stepper/stepper.cy.js

```diff
@@ -0,0 +1,13 @@
+describe("Test step content", () => {
+    it('test step content', () => {
+        cy.visit("/pages/layout/stepper");
+        cy.get('ngx-stepper > nb-card .step-content  .ng-star-inserted').as('position');
+        cy.get('@position').first().should('have.text', 'Step content #1');
+        cy.get('@position').last().click();
+        cy.get('@position').first().should('have.text', 'Step content #2');
+        cy.get('@position').last().click();
+        cy.get('@position').first().should('have.text', 'Step content #3');
+        cy.get('@position').last().click();
+        cy.get('@position').first().should('have.text', 'Step content #4');
```

Comment:
As option how to make it better
```suggestion
         const arrEl = ['Step content #1', 'Step content #2', 'Step content #3', 'Step content #4'];

        arrEl.forEach(el => {
            cy.get('ngx-stepper > nb-card .step-content  .ng-star-inserted').as('position')
            cy.get('@position').first().should('have.text', el);
            el === 'Step content #4' ? 'I finished'
                : cy.get('@position').last().click();
        })
```